Makefile

```diff
@@ -39,7 +39,13 @@ testf:
 	$(DOCKERRUN_NODE) node_modules/gulp/bin/gulp.js test:unit
 
 testb:
-	$(DOCKERRUN_PY) python -m unittest discover -s compair/tests/
+	$(DOCKERRUN_PY) nosetests
```

Comment:
The tests were running a bit slower than before so wanted to profile them to see (which nose does easily). 

See: 'testb-coverage' and 'testb-profile' in the make file